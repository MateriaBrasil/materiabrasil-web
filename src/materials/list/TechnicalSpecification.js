import React from 'react'
import CloudDownload from '@material-ui/icons/CloudDownload'
import Typography from '@material-ui/core/Typography'

export default ({ technicalSpecificationUrl }) => (
  <a
    href={technicalSpecificationUrl}
    target="_blank"
    style={{
      display: 'inline-block',
      textDecoration: 'none',
      marginRight: 24,
      color: 'white',
    }}
  >
    <CloudDownload
      color="inherit"
      style={{ verticalAlign: 'middle', marginRight: 8 }}
    />
    <Typography
      variant="body1"
      color="inherit"
      style={{
        display: 'inline-block',
        verticalAlign: 'middle',
      }}
    >
      Ficha t√©cnica
    </Typography>
  </a>
)
