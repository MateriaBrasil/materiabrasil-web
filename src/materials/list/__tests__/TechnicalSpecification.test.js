import React from 'react'
import renderer from 'react-test-renderer'

import TechnicalSpecification from '../TechnicalSpecification'

it('renders correctly', () => {
  const props = {
    technicalSpecificationUrl: 'foo-url',
    style: { foo: 'style' },
  }
  const tree = renderer.create(<TechnicalSpecification {...props} />).toJSON()
  expect(tree).toMatchSnapshot()
})
