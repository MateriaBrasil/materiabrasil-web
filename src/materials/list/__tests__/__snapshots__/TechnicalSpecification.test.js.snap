// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "foo": "style",
    }
  }
  tooltip="Baixar ficha tÃ©cnica"
  url="foo-url"
>
  FileDownloadIcon
</div>
`;
